### ISSUE TITLE
Hellblade - Senua's Sacrifice [Low Fps when using DX11 renderer]

### ISSUE THREAD

**Issue by jrugia** [doitsujin/dxvk#2611]
When using the DX11 renderer in this game I'm getting low fps, in some areas it even dips to 30's and 40's. I've tried with the DX12 renderer (vkd3d-proton v2.6) and the performance is much better. I've tried older dxvk versions, as far back as 1.7.x but the issue persists.

### Software information
Hellblade - Senua's Sacrifice, 2560x1440, "Very High" preset, Vsync off.

### System information
- GPU: RX 6800
- Driver: Mesa 22.0.1
- Wine version: 6.19 Staging (esync enabled)
- DXVK version: 1.10.1-async (also tried non async build)

### Log files
- d3d11.log: [HellbladeGame-Win64-Shipping_d3d11.log](https://github.com/doitsujin/dxvk/files/8496496/HellbladeGame-Win64-Shipping_d3d11.log)

- dxgi.log:  [HellbladeGame-Win64-Shipping_dxgi.log](https://github.com/doitsujin/dxvk/files/8496494/HellbladeGame-Win64-Shipping_dxgi.log)

Screenshot: 
![1](https://user-images.githubusercontent.com/13750621/163583980-0c462501-58b2-4bb1-b538-574c914d769d.jpg)

**Comment by doitsujin** [comment]
I've tested this game and had no such issues. You seem to be CPU-bound in Dx11 mode though.

**Comment by jrugia** [comment]
Hi, thanks for looking into it. I tried a newer version of Wine (7.0 to be exact) and the FPS is now similar to yours. Didn't change anything else except Wine version.

It seems my old Wine build (6.19) has an issue with bottlenecking the CPU, not only this game was performing badly due to it, but I've found out it was affecting others too, including Path of Exile's Vulkan renderer (non DXVK).

Closing ticket since issue has been resolved.
